stable/insights-agent/templates/kube-hunter/_job.yaml

```diff
@@ -25,6 +27,12 @@ containers:
   volumeMounts:
   - name: output
     mountPath: /output
+  - name: tmp
+    mountPath: /tmp
```

Comment:
why did we mount`/temp` here?

Reply:
In Trivy we had to do the same thing. kube-hunter must write files to /tmp, and when we make the root filesystem read only we can't write to tmp anymore if it's part of the root filesystem. But by creating an emptyDir volume (which is writeable) and mounting it to /tmp, then /tmp is writeable again.

Reply:
makes sense